catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/test/java/org/codice/alliance/catalog/plugin/defaultsecurity/DefaultSecurityAttributeValuesPluginTest.java

```diff
@@ -0,0 +1,202 @@
+
+    @Mock
+    private AttributeStatement mockedAttributeStatement2;
+
+    @Mock
+    private org.opensaml.saml.saml2.core.Attribute mockedAttribute2;
+
+    @Mock
+    private XSString mockedXss1;
+
+    @Mock
+    private XSString mockedXss2;
+
+    @Mock
+    private MetacardType metacardType;
+
+    @Mock
+    private AttributeDescriptor attributeDescriptor;
+
+    @Mock
+    private CreateRequest createRequest;
+
+    @Mock
+    private UpdateRequest updateRequest;
+
+    @Before
+    public void setup() {
+        initMocks(this);
+        defaultSecurityAttributeValuesPlugin =
+                Mockito.spy(new DefaultSecurityAttributeValuesPlugin());
+        doReturn(mockedSubject).when(defaultSecurityAttributeValuesPlugin)
+                .getSystemSubject();
+        when(mockedSubject.getPrincipals()).thenReturn(mockedPrincipalCollection);
+        when(mockedPrincipalCollection.oneByType(anyObject())).thenReturn(mockedAssertion);
+        when(mockedAssertion.getAttributeStatements()).thenReturn(Arrays.asList(
+                mockedAttributeStatement1,
+                mockedAttributeStatement2));
+        when(mockedAttributeStatement1.getAttributes()).thenReturn(Arrays.asList(mockedAttribute1));
+        when(mockedAttributeStatement2.getAttributes()).thenReturn(Arrays.asList(mockedAttribute2));
+        when(mockedXss1.getValue()).thenReturn("1");
+        when(mockedXss2.getValue()).thenReturn("2");
+        when(mockedAttribute1.getAttributeValues()).thenReturn(Arrays.asList(mockedXss1));
+        when(mockedAttribute2.getAttributeValues()).thenReturn(Arrays.asList(mockedXss2));
+        when(mockedAttribute1.getName()).thenReturn(USER_ATTRIB_1);
+        when(mockedAttribute2.getName()).thenReturn(USER_ATTRIB_2);
+        defaultSecurityAttributeValuesPlugin.setClassification(USER_ATTRIB_1);
+        defaultSecurityAttributeValuesPlugin.setCodewords(USER_ATTRIB_2);
+
+        when(markedMetacard.getAttribute(Metacard.SECURITY)).thenReturn(securityAttributeList);
+        when(securityAttributeList.getValues()).thenReturn(Collections.singletonList("blah"));
+        when(unmarkedMetacard.getMetacardType()).thenReturn(metacardType);
+        when(metacardType.getAttributeDescriptor(any(String.class))).thenReturn(attributeDescriptor);
+        when(createRequest.getMetacards()).thenReturn(Collections.singletonList(unmarkedMetacard));
+        when(updateRequest.getUpdates()).thenReturn(Collections.singletonList(Maps.immutableEntry(
+                "key",
+                unmarkedMetacard)));
+        when(emptySecurityAttributeList.getValue()).thenReturn(mockedEmptyAttributeMap);
+        when(mockedEmptyAttributeMap.isEmpty()).thenReturn(true);
+    }
+
+    @Test
+    public void testProcessUnmarkedMetacard() throws Exception {
+        Metacard modifiedMetacard = defaultSecurityAttributeValuesPlugin.addDefaults(
+                unmarkedMetacard);
+        assertThat(unmarkedMetacard, not(modifiedMetacard));
```

Comment:
no good reason I was originally only working with mocked objects and ripped out the verify calls and just forgot to add assertions when I changed it. fixed